ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -678,6 +682,9 @@ l2leaf:
     # Point-to-Point interface speed - will apply to L2 Leaf and L3 Leaf switches | Optional.
     p2p_link_interface_speed: < interface_speed >
 
+    # Enable / Disable auto MLAG, when two nodes are defined in node group.
```

Comment:
Should not we support this per node_group as well? Would be helpful in a mixed environment or even during design migration with MLAG and EVPN A/A

Reply:
Seems to be supported as per line 75 in `evpn-fabric-l3leaf-yml.j2`

Reply:
that's correct, maybe the documentation is clear, but all variables in `defaults` are also supported per node.
We have the following text on line 702 of README.md
```yaml
# All variables defined under `defaults` dictionary can be defined under each node group to override it.
```